x/claims/keeper/ibc_callbacks_test.go

```diff
@@ -576,10 +611,12 @@ func (suite *KeeperTestSuite) TestReceive() {
 
 				// check that the record is not deleted
 				suite.Require().True(suite.app.ClaimsKeeper.HasClaimsRecord(suite.ctx, secpAddr))
+				// TODO: check that IBC action is claimed
```

Comment:
address TODO

Reply:
@fedekunze I added the the EVM channel in `DefaultParams()` to the Authorized channels. Let me know if this is in line with the expected behaviour

Reply:
Nice catch! Will wait for Fede's answer.

I realize that we don't have tests for the case where the EVM channel is not in authorized channels.

Reply:
he ansered here: https://github.com/tharsis/evmos/pull/516#discussion_r854980824